apps/webapp/app/services/deleteProject.server.ts

```diff
@@ -0,0 +1,142 @@
+import { PrismaClient } from "@trigger.dev/database";
+import { prisma } from "~/db.server";
+import { DisableJobService } from "./jobs/disableJob.server";
+import { AuthenticatedEnvironment } from "./apiAuth.server";
+import { DeleteJobService } from "./jobs/deleteJob.server";
+import { DeleteEndpointService } from "./endpoints/deleteEndpointService";
+import { logger } from "./logger.server";
+import { DisableScheduleSourceService } from "./schedules/disableScheduleSource.server";
+
+type Options = { projectId: string; userId: string } | { projectSlug: string; userId: string };
```

Comment:
The slug variant is used at the moment. The id one will be used when deleting an org and iterating through the projects.